com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/OperationsHelper.java

```diff
@@ -107,6 +107,9 @@ public Operation addState(Operation operation) {
         if (flowableFacade.isProcessInstanceAtReceiveTask(processInstanceId)) {
             return Operation.State.ACTION_REQUIRED;
         }
+        if (flowableFacade.isProcessAborted(processInstanceId)) {
+            return Operation.State.ABORTED;
```

Comment:
Hmm, what about the cli? Could this result in error on CLI side?

Reply:
I've tested it and I haven't seen unexpected behavior. But now, we retry the process even though it was aborted